cirq/google/common_serializers.py

```diff
@@ -127,6 +127,54 @@ def _near_mod_2(e, t, atol=1e-8):
 ]
 
 
+#
+# PhasedXZ gate (de)serializer
+# TODO(maffoo): Move to SINGLE_QUBIT_(DE)SERIALIZERS when server support is in.
+#
+
+PHASED_X_Z_SERIALIZER = op_serializer.GateOpSerializer(
+    gate_type=ops.PhasedXZGate,
+    serialized_gate_id='xyz',
+    args=[
+        op_serializer.SerializingArg(
+            serialized_name='x_exponent',
+            serialized_type=float,
+            gate_getter='x_exponent',
+        ),
+        op_serializer.SerializingArg(
+            serialized_name='z_exponent',
+            serialized_type=float,
+            gate_getter='z_exponent',
+        ),
+        op_serializer.SerializingArg(
+            serialized_name='axis_phase_exponent',
+            serialized_type=float,
+            gate_getter='axis_phase_exponent',
+        ),
+    ],
+)
+
+PHASED_X_Z_DESERIALIZER = op_deserializer.GateOpDeserializer(
+    serialized_gate_id='xyz',
+    gate_constructor=ops.PhasedXZGate,
+    args=[
+        op_deserializer.DeserializingArg(
+            serialized_name='x_exponent',
+            constructor_arg_name='x_exponent',
+        ),
+        op_deserializer.DeserializingArg(
+            serialized_name='z_exponent',
+            constructor_arg_name='z_exponent',
+        ),
+        op_deserializer.DeserializingArg(
+            serialized_name='axis_phase_exponent',
+            constructor_arg_name='axis_phase_exponent',
+        ),
+    ],
+)
+
+
+
```

Comment:
nit: let's make it two empty lines for consistency

Reply:
Done.